app/views/orders/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<% if logged_in? %>
+  <% if @order.present? %>
+  <div class="order-container">
+    <h1><%= t("order.your_order") %></h1>
+    <% total_price = 0 %>
+    <% @order.each do |order_item| %>
```

Comment:
ủa aloo, order là 1 object mà sao foreach dc thế gái ơi

Reply:
Lúc em chạy thì em thấy for each được ạ. Vì em lưu order kiểu như mảng của các order_items